packages/backendai-ui/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "ES5",
```

Comment:
es5 is too old we should support ie or browsers running with es5? es2015 or 2016 would be find for modern browsers cc @yomybaby